src/impl/sbcl-vops.lisp

```diff
@@ -0,0 +1,292 @@
+;;;; sbcl-vops.lisp
```

Comment:
let's split this file. Maybe `sbcl-avx-vops` and `sbcl-x86-vops` to have AVX and PREFETCH respectively?

Reply:
Should I toss the package declaration in intrinsics-package.lisp or something like that?

Reply:
Moved to sbcl-intrinsics.lisp because it has sbcl-specific deps